### PR TITLE
[refactor] move code from AccountController to AccountService

### PR DESCRIPTION
**Pull request by Calvin979**
## What's changed?

As a layer of interaction with front-end, controller should not contain any code logic.

## Checklist

- [x]  I have read the [Contributing Guide](https://hertzbeat.apache.org/docs/community/code_style_and_quality_guide)
- [ ]  I have written the necessary doc or comment.
- [x]  I have added the necessary unit tests and all cases have passed.

## Add or update API

- [ ] I have added the necessary [e2e tests](https://github.com/apache/hertzbeat/tree/master/e2e) and all cases have passed.

### CODE REVIEW

**Review by tomsun28 — APPROVED** [review]
👍

File: manager/src/main/java/org/apache/hertzbeat/manager/service/AccountService.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import org.apache.hertzbeat.manager.pojo.dto.RefreshTokenResponse;
+
+/**
+ * Account service
+ */
+public interface AccountService {
+    Map<String, String> authGetToken(LoginDto loginDto) throws AuthenticationException;
+
+    RefreshTokenResponse refreshToken(String refreshToken) throws AuthenticationException;
+}
```
Comment (tomsun28) [inline manager/src/main/java/org/apache/hertzbeat/manager/service/AccountService.java:32]: hi can you help add code comments in interface method

Comment (Calvin979) [reply manager/src/main/java/org/apache/hertzbeat/manager/service/AccountService.java]: Code comments has added

**Comment by Calvin979** [comment]
@tomsun28 There are some controllers have the same problem like this pr.
Shall we create an issue for newcomers?

**Comment by tomsun28** [comment]
> @tomsun28 There are some controllers have the same problem like this pr. Shall we create an issue for newcomers?

hi, ok. 👍

**Review by tomsun28 — APPROVED** [review]

**Pull request merged**